package.json

```diff
@@ -10,34 +10,29 @@
     "tslint-config-shopify",
     "shopify"
   ],
-  "main": "base.json",
+  "main": "./base.json",
   "scripts": {
+    "prepublish": "yarn run build",
+    "build": "yarn run clean && tsc",
```

Comment:
I'd go with something closer to [`sewing-kit`'s build](https://github.com/Shopify/sewing-kit/blob/master/package.json), i.e.,:

- `build` compiles js=>ts, and sticks it in the `build` folder
- `dist` copies stuff into the module root
  - This eliminates ugly references to `dist`, and makes it a tad easier to visualize the final output

There's a bunch of extra files being included in the current build, and they're helping it to accidentally work.  Always preview the final product with `npm pack && tar -tf tslint-config-shopify-1.1.5.tgz`.